### ISSUE TITLE
kasan_symbolize: finds wrong function

### ISSUE THREAD

**Issue by dvyukov** [google/sanitizers#668]
Andrey Ryabinin noted a bug in kasan_symbolize.py:
https://groups.google.com/d/msg/syzkaller/Fu6BruqUHOU/dwSLNeHHAwAJ
```
No, it's a bug in your script. To find out source location, it uses 'function_name + offset' instead of absolute address. We have 2 kthread() functions in kernel and this confuses you script.

E.g. my vmlinux :
 $ addr2line -i -e vmlinux ffffffff811b5290
        /home/andrew/linux/kernel/kthread.c:178
 $ addr2line -i -e kasan_conf/vmlinux ffffffff825c7240
        /home/andrew/linux/drivers/block/aoe/aoecmd.c:1289

 $ echo '[<ffffffff811b5290>] kthread+0x00/0x00' | python kasan_symbolize.py vmlinux
        [<ffffffff811b5290>] kthread+0x00/0x00 drivers/block/aoe/aoecmd.c:462
 $ echo '[<ffffffff825c7240>] kthread+0x00/0x00' | python kasan_symbolize.py vmlinux
        [<ffffffff825c7240>] kthread+0x00/0x00 drivers/block/aoe/aoecmd.c:462
```

**Comment by dvyukov** [comment]
@xairy @ramosian-glider need somebody with python-fu

**Comment by dvyukov** [comment]
We could look at function size to choose the right one.

**Comment by ramosian-glider** [comment]
Fixed by bf3fc6c4071e9bcdce5ba14119dbd8cf58b1b4d1
